src/main/java/com/checkmarx/flow/service/ConfigurationOverrider.java

```diff
@@ -45,7 +46,7 @@
 
     public ScanRequest overrideScanRequestProperties(CxConfig override, ScanRequest request) {
         ConfigProvider configProvider = ConfigProvider.getInstance();
-        if (request == null || (!configProvider.hasAnyConfiguration(MDC.get("cx")) && (override == null || Boolean.FALSE.equals(override.getActive())))) {
+        if (request == null || (!isConfigAsCodeAvailable(configProvider) && isLegacyConfigAsCodeAvailable(override))) {
```

Comment:
Shouldn't it be `!isLegacyConfigAsCodeAvailable(override)`?

Reply:
I changed it to 'isLegacyConfigAsCodeNotAvailable'